zerver/models.py

```diff
@@ -483,6 +483,7 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
 
     USERNAME_FIELD = 'email'
     MAX_NAME_LENGTH = 100
+    NAME_INVALID_CHARS = ['*', '`', '\'', '>', '\"', '@']
```

Comment:
Just a cosmetic thing, but you can avoid the backslashes by saying `"'"` and `'"'`.